README.md

```diff
@@ -2,4 +2,21 @@
 Electrium Mobility Discord Bot Developing...
 
 # Project Setup
-![Alt text](image.png)
\ No newline at end of file
+![Alt text](image.png)
```

Comment:
I think this overlaps with Carl's changes, might need to redo readme

Reply:
Ill create another pull to revert the changes